automotive/maliput/utility/generate_obj.cc

```diff
@@ -382,32 +649,59 @@ void StripeLaneBounds(GeoMesh* mesh, const api::Lane* lane,
   const double half_stripe = 0.5 * stripe_width;
 
   const double s_max = lane->length();
+
+  GeoFace prev_left_face;
+  GeoFace prev_right_face;
+
   for (double s0 = 0; s0 < s_max; s0 += grid_unit) {
     double s1 = s0 + grid_unit;
     if (s1 > s_max) { s1 = s_max; }
 
     api::RBounds rb0 = lane->lane_bounds(s0);
     api::RBounds rb1 = lane->lane_bounds(s1);
 
-    // Left side of lane.
+    // Left side of lane
```

Comment:
I believe this is an unintentional change.

Reply:
Indeed, thanks for catching that.